### DISCUSSION TITLE
IGPU passthrough on Proxmox 8.1?

### DISCUSSION THREAD

**Discussion by mr-suw** [isc30/ryzen-gpu-passthrough-proxmox#12] [Q&A] [answered]
Hi, did anyone upgrade to Proxmox 8.1 and has the IGPU passthrough working for a windows 10 VM? Any re-configuration required to get it working?

Thanks for the guideline to get following IGPU working on release 8.0 of Proxmox:
- Win 10 VM with IGPU of AMD Ryzen 9 7940HS w/ Radeon 780M working in 8.0.

My issue after upgrade to 8.1 is as follows:
VM starts and whole system gets stuck. Could not find any good log output. 
Removing IGPU devices from VM and switching graphics HW to standard VGA gets the VM running without GPU acceleration.

Where should I look into to get better log output?

**Comment by isc30** [comment]
Hey, yes I'm on `pve v8.1.3` and it all works as usual. I would start with the following:
- make the VM **not** start at boot
- reboot the host
- check with `lspci -nnk` that all the host drivers are `vfio-pci`
- start the vm

**Comment by mr-suw** [comment]
It's great to read that v8.1.3 is working for you. 
I rebooted and checked my configuration. The kernel driver for the AMD audio and vga device are set to vfio-pci.
Figured out by running `update-initramfs -u`that system seems to be booted in EFI-mode (bios support for that is off). Message was: System booted in EFI-mode but 'grub-efi-amd64' meta-package not installed!
I installed it, rerun update-initramfs and rebooted. 
Unfortunately the system is still stuck after running VM.

Further suggestions are welcome.

**Comment by mr-suw** [comment, marked as answer]
I could get it working again in pre 8.1.x. Just make sure you also passthrough as pcie the following AMD device: ACP/ACP3X/ACP6x Audio Coprocessor 
Please make sure that the host does not use its sound driver (blacklist driver and make a softdep configuration)
